tweb/src/main/frontend/src/js/vue/components/ProjectCard.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+	<div class="project-container">
+		<div class="project-icon">
+			<img v-bind:src="project.logoLocation">
+		</div>
+		<a v-bind:href="project.projectLink">
+			<div class="project-request-details" :style="{ 'background': project.overviewColor }">
+				<div class="project-request-name"><h3>{{ project.name }}</h3></div>
+				<div class="project-request-description"><p>{{ project.description }}</p></div>
+				<hr />
+				<div class="requestCountContainer">
+					<div class="requestNumber"><p>#{{ boundedRequests }}</p></div>
+					<div class="requestLabel"><p>active</p><p>requests</p></div>
+				</div>
+			</div>
+		</a>
+	</div>
+</template>
+
+<script lang="ts">
+    import {Component, Prop, Vue} from "vue-property-decorator";
+	import ProjectOverviewDetail from "../models/ProjectOverviewDetail"
+
+    @Component
+    export default class ProjectCard extends Vue {
+
+		defaultProject: ProjectOverviewDetail = {
+			name: "FundRequest",
+			description: "Marketplace for Software Development",
+			overviewColor: "#000000",
+			projectLink: "https://fundrequest.io/requests?",
+			logoLocation: "/assets/img/logo-single180.png",
+			activeRequests: 5
+		};
+
+		@Prop({ 
+			default: function() { 
+				return {
+					name: "FundRequest",
+					description: "Marketplace for Software Development",
+					overviewColor: "#000000",
+					projectLink: "https://fundrequest.io/requests?",
+					logoLocation: "/assets/img/logo-single180.png",
+					activeRequests: 5
+				};
+			} 
+		}) project: ProjectOverviewDetail
+
+		get colorStyle() {
+			return {
+				'background' : this.project.overviewColor
+			}
+		}
+
+		get boundedRequests() {
+			if (this.project.activeRequests > 99) {
+				return "99+";
+			} else {
+				return this.project.activeRequests;
+			}
+		}
+
+		private _mergeWithDefault() {
+			this.project.name = this.project.name || this.defaultProject.name;
```

Comment:
Merging can be done with
`this.project = Object.assign(this.defaultProject, this.project)`;
You can still do 
```
if (this.project.logoLocation !== this.defaultProject.logoLocation) {
    this.project.logoLocation = this.project.logoLocation + '?size=150';
}
```